code/xcvm/cosmwasm/contracts/gateway/src/contract/execute.rs

```diff
@@ -60,7 +60,12 @@ pub fn execute(deps: DepsMut, env: Env, info: MessageInfo, msg: msg::ExecuteMsg)
 	}
 }
 
-fn handle_config_msg(auth: auth::Admin, deps: DepsMut, msg: ConfigSubMsg, env: Env) -> Result {
+fn handle_config_msg(
+	auth: auth::Admin,
+	deps: &mut DepsMut,
```

Comment:
This should still work with plain `mut deps: DepsMut`, no? This will leave more code unchanged.